### DISCUSSION TITLE
Help needed for JSON Config for Linknode R4

### DISCUSSION THREAD

**Discussion by pbogaert** [RavenSystem/esp-homekit-devices#1577] [MEPLHAA Scripts]
Hi All,

Has anyone done a HAA on a linknode R4?

4 relays on a ESP-12F/ESP8266

R1 on 12
R2 on 13
R3 on 14
R4 on 16

I've added 2 buttons on GPIO 4 and 5 as well.

I want to use this as a sprinkler for 4 zones.

I've tried a couple of times, but can't get it to work :-(

**Comment by Hanimaniac** [comment]
I use a Shelly RGBW2. Just change the gpio's you need.

4 Water Valves (Sprinkler)
Setup -& Update Switch

{"c":{"l":13,"b":[{"g":0,"t":5}]},
"a":[{"0":{"r":[{"g":12,"v":0}]},"1":{"r":[{"g":12,"v":1}]},
"t":20,"s":0,"w":1,"d":3600,"f0":[]},{"0":{"r":[{"g":15,"v":0}]},"1":
{"r":[{"g":15,"v":1}]},
"t":20, "s":0,"w":1,"d":3600,"f0":[]},{"0":{"r":[{"g":14,"v":0}]},"1":
{"r":[{"g":14,"v":1}]},
"t":20, "s":0,"w":1,"d":3600,"f0":[]},{"0":{"r":
[{"g":4,"v":0}]},"1":{"r":[{"g":4,"v":1}]},
"t":20,"s":0,"w":1,"d":3600,"f0":[]},{"1":{"s":[{"a":1}]}},{"1":{"s":[{"a":2}]}}]}
